regl-camera.js

```diff
@@ -41,10 +42,14 @@ function createCamera (regl, props_) {
   var prevY = 0
 
   if (isBrowser && props.mouse !== false) {
-    mouseChange(function (buttons, x, y) {
+    var source = element || window
+    var width = element ? element.offsetWidth : window.innerWidth
+    var height = element ? element.offsetHeight : window.innerHeight
+    var noScroll = true
```

Comment:
This *could* be optional though that's maybe unnecessary. Nice to have it verbose at least so you don't have to guess what the `true` does. 👍